common/src/main/resources/assets/voicechat/lang/es_es.json

```diff
@@ -1,213 +1,217 @@
-  "cloth_config.voicechat.config.audio_packet_threshold": "Umbral de paquetes de audio",
-  "cloth_config.voicechat.config.audio_packet_threshold.description": "La cantidad máxima de paquetes de audio que se retienen si un paquete llega fuera de orden o se cae.\nAsí se evita descartar paquetes de audio ligeramente desordenados\nPóngalo a 0 para desactivarlo",
-  "cloth_config.voicechat.config.voice_deactivation_delay": "Retardo de desactivación",
-  "cloth_config.voicechat.config.voice_deactivation_delay.description": "El tiempo que tarda el micrófono en desactivarse cuando se utiliza la activación por voz.\nUn valor de 1 significa 20 milisegundos, 2=40 ms, 3=60 ms, …",
-  "cloth_config.voicechat.config.output_buffer_size": "Tamaño del búfer de salida",
-  "cloth_config.voicechat.config.output_buffer_size.description": "El tamaño del búfer de salida de audio en paquetes\nValores más altos significan una mayor latencia, pero menos crujidos\nAumente este valor si su conexión a Internet es inestable.",
-  "cloth_config.voicechat.config.hud_icon_scale": "Escala del icono HUD",
-  "cloth_config.voicechat.config.hud_icon_scale.description": "La escala de los iconos del HUD",
-  "cloth_config.voicechat.config.hud_icon_pos_x": "Icono HUD posición X",
-  "cloth_config.voicechat.config.hud_icon_pos_x.description": "La posición X de los iconos del HUD\nLos valores negativos significan anclaje a la derecha",
-  "cloth_config.voicechat.config.hud_icon_pos_y": "Icono HUD posición Y",
-  "cloth_config.voicechat.config.hud_icon_pos_y.description": "La posición Y de los iconos del HUD\nLos valores negativos significan anclaje a abajo",
-  "cloth_config.voicechat.config.group_player_icon_orientation": "Orientación del icono HUD de grupo",
-  "cloth_config.voicechat.config.group_player_icon_orientation.description": "La orientación de los iconos de los jugadores en el HUD del grupo.",
-  "cloth_config.voicechat.config.group_hud_icon_scale": "Escala del icono HUD del grupo",
-  "cloth_config.voicechat.config.group_hud_icon_scale.description": "La escala del HUD del grupo",
-  "cloth_config.voicechat.config.group_player_icon_pos_x": "Icono de grupo Posición X",
-  "cloth_config.voicechat.config.group_player_icon_pos_x.description": "Posición X de los iconos de los jugadores en el HUD del grupo\nLos valores negativos significan anclaje a la derecha",
-  "cloth_config.voicechat.config.group_player_icon_pos_y": "Icono de grupo Posición Y",
-  "cloth_config.voicechat.config.group_player_icon_pos_y.description": "Posición Y de los iconos de jugador en el HUD de grupo\nLos valores negativos significan anclaje a abajo",
-  "cloth_config.voicechat.config.show_own_group_icon": "Mostrar icono HUD del grupo propio",
-  "cloth_config.voicechat.config.show_own_group_icon.description": "Si se debe mostrar el icono propio cuando se está en un grupo"
+  "message.voicechat.onboarding.introduction.title": "Asistente de Configuración de Simple Voice Chat",
```

Comment:
Why did you hardcode the mod name? This translation takes one argument thats the mod name.
This translation also does not convey the same information as the english translation.